src/main/java/org/yaml/snakeyaml/introspector/PropertyUtils.java

```diff
@@ -65,7 +67,7 @@ protected Map<String, Property> getPropertiesMap(Class<?> type, BeanAccess bAcce
     }
 
     Map<String, Property> properties = new LinkedHashMap<String, Property>();
-    boolean inaccessableFieldsExist = false;
+    boolean[] inaccessableFieldsExist = {false};
```

Comment:
this is changed to array to mutate the data. This should be at least **documented**. 
(I wonder if there is a better solution...)

Reply:
Good point. The `boolean[1]` return value pattern has no reason to be used in this case. Refactored in 9f0df9b